runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridPlugin.java

```diff
@@ -38,6 +46,8 @@
 	tags = {"items", "overlay"},
 	enabledByDefault = false
 )
+
+@Slf4j
```

Comment:
Are you logging anything in this plugin? I don't see this being used.

Reply:
It was for testing, I will remove.